src/environments/environment.prod.ts

```diff
@@ -52,4 +52,14 @@ export const environment: Environment = {
       message: 'entry.971826736',
     },
   },
+  merchForm: {
+    url: 'https://docs.google.com/forms/d/e/1FAIpQLSf3Aet204vFeXXP5IkWm0oEK_rm-VTvHFl6mwEXQxwuounRSg/formResponse',
```

Comment:
Can you create a separate form for receiving live orders?
Just thinking it'd make it easier distinguish real submissions from test submissions.

Reply:
Fixed in d6b77705f30f9c6a1e68cd00d3ff467951dcc149